app/Http/Controllers/Achievements/AchievementsController.php

```diff
@@ -31,7 +33,7 @@ public function __construct(TCAPI $api)
      *
      * @param Request $request
      * @param int $id
-     * @return Achievement|array|\Illuminate\Database\Eloquent\Collection|\Illuminate\Database\Eloquent\Model|\Illuminate\Database\Query\Builder|static[]
+     * @return Achievement|array
      */
     public function getAchievements(Request $request, $id = 0) {
```

Comment:
You don't need a Request object as argument. 
The global function `request()` returns this object.